ecosystem/sep-0030.md

```diff
@@ -16,6 +16,10 @@ Version: 0.8.0
 
 This protocol defines an API that enables an individual (e.g., a user or wallet) to regain access to a Stellar account that it owns after the individual has lost its private key without providing any third party control of the account. Using this protocol, the user or wallet will preregister the account and a phone number, email, or other form of authentication with one or more servers implementing the protocol and add those servers as signers of the account. If two or more servers are used with appropriate signer configuration no individual server will have control of the account, but collectively, they may help the individual recover access to the account. The protocol also enables individuals to pass control of a Stellar account to another individual.
 
+## Dependencies
+
+Implementers and clients of this protocol must also implement [SEP-10].
```

Comment:
```suggestion
## Dependencies

Implementers and clients of this protocol must also implement [SEP-10](sep-0010.md).
```

Reply:
The `(sep-0010.md)` is unnecessary. The `[SEP-10]` link is defined in a footnote.